OpenStack-accounting/usr/local/sbin/cinder_extract_accounting.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+import accountinglib
+
+import sys
+import time
+import datetime
+
+
+nowtime = time.localtime()
+
+print("Cinder Accounting run start")
+starttime=sys.argv[1]
+print("Start Time = " + starttime)
+endtime=sys.argv[2]
+print("End Time = " + endtime)
+endyyyymm=datetime.datetime.strptime(endtime,"%Y-%m-%d %H:%M").strftime('%Y-%m')
+endtimestamp = time.mktime(datetime.datetime.strptime(endtime, "%Y-%m-%d %H:%M").timetuple())
+
+
+results = accountinglib.get_accounting_data("cinder",starttime,endtime)
+datastring = ''
+print(results)
+for result in results:
+    print(result)
+    try:
+        if "rally" in result["Project"] :
+            department="STFC Cloud"
+        else:
+            if "default" in result["Department"]:
+                department=result["Project"]
+            else:
+                department=result["Department"]
+
+    except:
+        department="UNKNOWN"
```

Comment:
```suggestion
if "rally" in result.get("Project", ""):
    department = "STFC Cloud"
elif "default" in result.get("Department", ""):
    department = result.get("Project", "")
else:
    department = result.get("Department", "UNKNOWN")
```
try catch here is not necessary and could lead to bugs in the python code not being caught in the code block